[OSDOCS#9208]: Docs work for IPI: ShiftStack dual stack support (#71011)
openshift/openshift-docs: modules/install-osp-deploy-dualstack.adoc

. Add the IPv6 subnet to the router to enable router advertisements. If you are using a provider network, you can enable router advertisements by adding the network as an external gateway, which also enables external connectivity.

. To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `install-config.yaml` file. The following is an example of an `install-config.yaml` file.

> .Example `install-config.yaml`

Reviewer: should this be kept?

Author: good catch! ty!

> . To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `install-config.yaml` file.

Reviewer: line 27 and 28 are leftover

> The following is an example of an `install-config.yaml` file for an IPv4 primary dual-stack install-config.

Reviewer: ```suggestion
To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `install-config.yaml` file. The following is an example of an `install-config.yaml` file.
```

> To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `install-config.yaml` file. The following is an example of an `install-config.yaml` file.

Reviewer: do you need a dot int the beginning of this sentence?

Author: fixed :) I had an extra repeated line

. Add the IPv6 subnet to the router to enable router advertisements. If you are using a provider network, you can enable router advertisements by adding the network as an external gateway, which also enables external connectivity.

. To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `install-config.yaml` file.
+

> The following is an example of an `install-config.yaml` file:

Reviewer: nit: I think this should be a continuation of line 27 and not a new paragraph

Author: done!

    hostPrefix: 64
  serviceNetwork: <1>
  - 172.30.0.0/16
  - fd02::/112
platform:
  openstack:
    ingressVIPs: ['192.168.25.79', 'fd2e:6f44:5dd8:c956:f816:3eff:fef1:1bad'] <2>
    apiVIPs: ['192.168.25.199', 'fd2e:6f44:5dd8:c956:f816:3eff:fe78:cf36'] <3>
    controlPlanePort: <4>
      fixedIPs: <5>
      - subnet: <6>
          name: subnet-v4
          id: subnet-v4-id
      - subnet: <6>
          name: subnet-v6
          id: subnet-v6-id
      network: <7>
        name: dualstack
        id: network-id
----
<1> You must specify an IP address range for both IPv4 and IPv6 address families.
<2> Specify the virtual IP (VIP) address endpoints for the Ingress VIP services to provide an interface to the cluster.
<3> Specify the virtual IP (VIP) address endpoints for the API VIP services to provide an interface to the cluster.
<4> Specify the dual-stack network details that are used by all the nodes across the cluster.
<5> The CIDR of any subnet specified in this field must match the CIDRs listed on `networks.machineNetwork`.
<6> You can specify a value for either `name` or `id`, or both.
<7> Specifying the `network` under the `ControlPlanePort` field is optional
+

> The following is an example of an `install-config.yaml` file for an example of an IPv6 primary dual-stack install-config.

Reviewer: ```suggestion
To configure an IPv6 primary dual-stack cluster, edit the `install-config.yaml` file. The following is an example of an `install-config.yaml` file.
```

> . Alternatively if you want an IPv6 primary dual-stack cluster, edit the `install-config.yaml` file following the example below:

Reviewer 2: ```suggestion
. Alternatively, if you want an IPv6 primary dual-stack cluster, edit the `install-config.yaml` file following the example below:
```

Reviewer 2: If the user performs step 4 or step 5, should they be non-numbered substeps under the same step? 

```
Perform one of the following: 
* To configure IPv4 and IPv6 address endpoints for cluster nodes....
* To configure an IPv6 primary dual-stack cluster... 

```
If they are sequential steps, maybe use _Optional:_.

Author: Ah - good call. They are not sequential, so I removed the numbering so that it still falls under step 4. :)

> To configure an IPv6 primary dual-stack cluster, edit the `install-config.yaml` file. The following is an example of an `install-config.yaml` file.

Reviewer: might worth to reformulate it to the following:

```suggestion
. Alternatively if you want an IPv6 primary dual-stack cluster, edit the `install-config.yaml` file following the example bellow:
```
